oaknational/Oak-Web-Application
# Feat/lesq 975/sign out button [LESQ-975]

## Description

Music year: 2007

- adds a sign out button to the hamburger menu when a user is signed in
- it is not behind the feature flag as I didn't think it was necessary when you can only sign in behind the feature flag - lmk if you disagree on this

## How to test

1. Go to https://deploy-preview-2675--oak-web-application.netlify.thenational.academy (any page) while signed in
2. Click on the hamburger menu
3. You should see the sign out button
4. Click on sign out and reopen the hamburger menu
5. You should not see the sign out button
6. When the feature flag is not active and you are not signed in you should not see the sign out button

## Screenshots
How it should now look:
<img width="600" alt="Screenshot 2024-08-07 at 09 04 17" src="https://github.com/user-attachments/assets/6e4a3692-7219-4e82-905c-38f2245d20df">


## Checklist

- [ ]  A button in the ~~navbar~~ hamburger menu should show when a user is logged in
- [ ]  When no user is logged in there is no button
- [ ]  The sign out button should appear across OWA products - curriculum, pupils, teachers etc
